src/main/scala/org.rogach.scallop/ScallopConfBase.scala

```diff
@@ -616,6 +616,46 @@ abstract class ScallopConfBase(
       .getOrElse(Right(()))
   }
 
+  def validateFilesExists(filesOption: ScallopOption[List[File]]) = addValidation {
+    filesOption.toOption
+	  .map(files => {
```

Comment:
Can you please replace tabs with two spaces? Currently tabs mess up the formatting.